components/x-live-blog-wrapper/src/__tests__/LiveBlogWrapper.test.jsx

```diff
@@ -74,6 +74,19 @@ describe('liveBlogWrapperActions', () => {
 		expect(posts[0].id).toEqual('3')
 	})
 
+	it('does not insert a new post if a duplicate', () => {
+		const post2 = {
```

Comment:
I'm going to change this logic so it intentionally duplicates an element in the original posts array